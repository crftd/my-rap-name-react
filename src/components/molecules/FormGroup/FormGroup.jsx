import React from 'react';
import styled from 'styled-components';

const FormGroupWrapper = styled.div`
  min-height: 70px;
  padding-bottom: 10px;
`;

export default class FormGroup extends React.Component {
  render() {
    return <FormGroupWrapper>{this.props.children}</FormGroupWrapper>;
  }
}
