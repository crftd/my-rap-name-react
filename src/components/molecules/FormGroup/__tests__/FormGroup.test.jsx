import React from 'react';
import { shallow } from 'enzyme';
import FormGroup from '../FormGroup.jsx';

describe('[Snapshot] FormGroup', () => {
  it('should render', () => {
    // Arrange

    // Act
    const actualTree = shallow(<FormGroup />);
    // Assert
    expect(actualTree).toMatchSnapshot();
  });
});
