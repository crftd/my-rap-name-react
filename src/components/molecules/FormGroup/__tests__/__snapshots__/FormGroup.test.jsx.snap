// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[Snapshot] FormGroup should render 1`] = `<styled.div />`;
